classes/app.rb

```diff
@@ -5,10 +5,11 @@
 class App
   def initialize
     @items = []
+    @labels = []
 
     @manage_items = ManageItems.new(@items)
-    @manage_labels = ManageLabels.new(@items)
-    @manage_genres = ManageGenres.new(@items)
+    @manage_labels = ManageLabels.new(@labels)
+    @manage_genres = ManageGenres.new(@labels)
```

Comment:
We need a variable for gener as well. `@genres` but you're passing the `@labels`

Reply:
@aimalamiri, Thanks for pointing this issue out. I will implement this right away.